src/lib.rs

```diff
@@ -145,12 +101,8 @@ pub use primitives::*;
 pub use promise::{PromiseRejectEvent, PromiseRejectMessage, PromiseState};
 pub use property_attribute::*;
 pub use proxy::*;
-pub use scope::CallbackScope;
-pub use scope::ContextScope;
-pub use scope::FunctionCallbackScope;
-pub use scope::PropertyCallbackScope;
-pub use scope::Scope;
-pub use scope_traits::*;
+#[doc(inline)]
+pub use scope::*;
```

Comment:
I would prefer if you explicitly listed exactly what is made public. [Elsewhere](https://github.com/denoland/deno/blob/0ffc99a61ddfa958a436beef0d003ecead630d0f/core/lib.rs#L26-L50) we've found it helpful to not use `pub use` wildcard statements.

```rust
pub use scope::api::{CallbackScope, ContextScope, EscapableHandleScope, HandleScope};
```